docs/source/deployment/index.rst

```diff
@@ -169,3 +169,144 @@ Almost all functionalities available in MLflow deployment can also be accessed v
 
 FAQ
 ---
+
+How to test my model before deploying to the production environment?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Testing your model before deployment is a critical step to ensure production readiness.
+MLflow provides a few ways to test your model locally, either in a virtual environment or a Docker container.
+
+Testing offline prediction with a virtual environment
+*****************************************************
+You can use MLflow `predict` API via CLI or Python to making test predictions with your model.
```

Comment:
```suggestion
You can use `mlflow models predict` API via CLI or Python to making test predictions with your model.
```

Perhaps it's just me, but `predict` API sounds ambiguous. Can we use the full command name?

Reply:
Makes sense to me, will replace:)